auto-update-pip-packages/README.md

```diff
@@ -6,9 +6,9 @@
 
 Install the cron job. As root:
 
-```bash
-# cp cron /etc/cron.d/openfisca-auto-update-pip-packages
-# systemctl restart cron
+```sh
```

Comment:
By curiosity, what's the reason changing from bash to sh? To be more general or to match a list of syntax highlighters?